p2p/agent.go

```diff
@@ -495,8 +495,7 @@ func (p *agent) joinP2P(ctx context.Context) error {
 	if err := p.host.Advertise(); err != nil {
 		return err
 	}
-	p.host.FindPeers(ctx)
-	return nil
+	return p.host.FindPeers(ctx)
```

Comment:
The error is not returned on purpose. Because peers can't be found immediately after advertising. But logging could be added instead of doing nothing

Reply:
ok. fixed